renderer/types/actions/field-actions.ts

```diff
@@ -0,0 +1,61 @@
+import * as consts from '../consts';
+
+export interface FieldActions {
+  updateTimer: (msg: {
+    total_stage_time: number;
+    stage_time_so_far: number;
+    stage_name: string;
+  }) => {
+    type: consts.FieldActionsTypes.UPDATE_TIMER;
+    timeLeft: number;
+    stage: string;
+    totalTime: number;
+  };
+
+  updateHeart: () => {
+    type: consts.FieldActionsTypes.UPDATE_HEART;
+  };
+
+  updateMaster: (msg: {
+    blue: string;
+    gold: string;
+  }) => {
+    type: consts.FieldActionsTypes.UPDATE_MASTER;
+    blueMaster: string;
+    goldMaster: string;
```

Comment:
May I ask why are these string? I'm looking at `StatusLabel.tsx` and in the propTypes it says `blueMaster: PropTypes.number.isRequired`.

Reply:
@HaileyJang Hmm, you're right, I put the wrong type for those. From the name, it's actually a bit unclear, so I think let's change it to `blueMasterTeamNumber` and `goldMasterTeamNumber`.